### PR TITLE
test(scanner): Improve tests for `ScannerWrapperConfig`

### PR DESCRIPTION
**Pull request by mnonnenmacher**

### CODE REVIEW

File: scanner/src/test/kotlin/ScannerWrapperConfigTest.kt [hunk 1]
```diff
@@ -59,5 +60,24 @@ class ScannerWrapperConfigTest : WordSpec({
...
+
+        "set unset storage options to null" {
+            with(ScannerWrapperConfig.create(emptyMap()).first) {
+                readFromStorage shouldBe null
+                writeToStorage shouldBe null
+            }
+        }
+
+        "fail if storage options are invalid" {
+            shouldThrow<IllegalArgumentException> {
```
Comment (fviernau) [inline scanner/src/test/kotlin/ScannerWrapperConfigTest.kt:72]: consider

```
listOf("1", "tru", "null").forAll {
    shouldThrow<IllegalArgumentException> {
          ScannerWrapperConfig.create(mapOf(ScannerWrapperConfig.PROP_READ_FROM_STORAGE to it))
    }
}

File: scanner/src/test/kotlin/ScannerWrapperConfigTest.kt [hunk 2]
```diff
@@ -59,5 +60,24 @@ class ScannerWrapperConfigTest : WordSpec({
 
             ScannerWrapperConfig.create(options).second shouldContainExactly mapOf("other" to "value")
         }
+
+        "set unset storage options to null" {
```
Comment (fviernau) [inline scanner/src/test/kotlin/ScannerWrapperConfigTest.kt:64]: I don't understand the test case name. Mind giving a hint?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: scanner/src/test/kotlin/ScannerWrapperConfigTest.kt (hunk 2 above)
Comment (sschuberth) [reply scanner/src/test/kotlin/ScannerWrapperConfigTest.kt:64]: I guess you should read it like "storage option that are unset in the config should be explicitly set to null in the class properties".

Comment (mnonnenmacher) [reply scanner/src/test/kotlin/ScannerWrapperConfigTest.kt]: Is the new name better?

**Review by fviernau — APPROVED** [review]

**Pull request merged**